community-plugins.json

```diff
@@ -4759,7 +4759,7 @@
         "id": "obsidian-party",
         "name": "Party🎉",
         "author": "Shap Po",
-        "description": "An implementation of party.js for Obsidian. Create confetti & sparkles in your notes!",
+        "description": "An implementation of party.js for Obsidian. Create confetti, sparkles and even custom effects in your notes!",
```

Comment:
"Create confetti, sparkles, and even custom effects in your notes!" is probably better? (have an extra comma)